gpflow/ci_utils.py

```diff
@@ -18,7 +18,17 @@
 
```

Comment:
Pls, move this file to notebooks folder

Reply:
That would require changing all the notebooks. Let's do this once this branch is in.

Reply:
You can't move it to the notebooks folder because then you can't import it in the notebooks without hacking the `sys.path`, which is super hacky.